dao/src/main/kotlin/repositories/ortrun/DaoOrtRunRepository.kt

```diff
@@ -152,6 +152,12 @@ class DaoOrtRunRepository(private val db: Database) : OrtRunRepository {
         }.map { ActiveOrtRun(it.id.value, it.createdAt, it.traceId) }
     }
 
+    override fun findRunsBefore(before: Instant): List<Long> = db.blockingQuery {
```

Comment:
Instead of adding a new function, could you maybe replace `listRunsBefore` instead to avoid dead code ?

Reply:
Yes, I was not sure whether there was a use case for the existing function - maybe for the UI, so that users can see older runs that are going to be removed soon? But I am also fine with replacing it. Should the function then be named `listRunsBefore`? Since the `list` functions typically return entities.

Reply:
Yes, fine with me, or `listRunsIdsBefore`. Or `findRunsIdsBefore`. As you want :)

Reply:
I replaced `listRunsBefore` by `findRunsBefore`.